main.py

```diff
@@ -23,6 +25,7 @@ def setup(self):
         """Initialize the NApp's links list."""
         self.links = {}
         self.store_items = {}
+        self.link_up_timer = settings.LINK_UP_TIMER
```

Comment:
Maybe it would be better to have a default value here? This way the code won't break if a value is missing from the configuration.

```suggestion
        self.link_up_timer = getattr(settings, 'LINK_UP_TIMER', 10)
```

Reply:
You're right. It's done.